superset-frontend/src/components/DeleteModal.tsx

```diff
@@ -65,7 +65,7 @@ export default function DeleteModal({
     >
       <DescriptionContainer>{description}</DescriptionContainer>
       <StyleFormGroup>
-        <FormLabel htmlFor="delete">{t('type "delete" to confirm')}</FormLabel>
+        <FormLabel htmlFor="delete">{t('Type "DELETE" to confirm')}</FormLabel>
```

Comment:
Maybe make "DELETE" a parameter `t('Type "%s" to confirm', t('DELETE'))`?

Reply:
Ah, perfect! — will do.